lib/cf/ServiceFabrikClient.js

```diff
@@ -67,7 +67,11 @@ class ServiceFabrikClient extends HttpClient {
       .then(accessToken => this
         .request({
           method: 'DELETE',
-          url: `/api/v1/backups/${options.backup_guid}?space_guid=${options.tenant_id}`,
+          url: `/api/v1/backups/${options.backup_guid}`,
+          qs: {
+            space_guid: options.tenant_id,
+            only_snapshot: options.only_snapshot
```

Comment:
API should not take such params. External consumers must not be aware of the internal backup mechanisms that are done by SF and services. Having such a flag spills the logic outside, which is not desirable.